exporter/exporterhelper/internal/queuebatch/multi_batcher.go

```diff
@@ -36,24 +36,17 @@ type multiBatcher struct {
 var _ Batcher[request.Request] = (*multiBatcher)(nil)
 
 func newMultiBatcher(bCfg BatchConfig, bSet batcherSettings[request.Request]) *multiBatcher {
-	var workerPool chan struct{}
-	if bSet.maxWorkers != 0 {
```

Comment:
maxWorkers cannot be 0, because https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/internal/queuebatch/config.go#L77